### ISSUE TITLE
Bug - Text area - content overflows focus ring

### ISSUE THREAD

**Issue by srambach** [patternfly/patternfly-react#10883]
**Describe the problem**
This is an issue with 
- [ ] Patternfly 5  
- [x] Patternfly 6  
- [ ] other

**How do you reproduce the problem?**
Put enough text into the text area to cause it to grow vertically. While the text area is focused, you can see that that the focus ring overlaps the content in the vertical direction.

**Expected behavior**
Ideally, the text should be contained within the focus ring area, which is inside the boundary of the text input. 
Note that the auto-resizing version of the text area does not seem to have this issue.

**Screenshots**
<img width="845" alt="image" src="https://github.com/user-attachments/assets/8265cdd0-1f9c-4bf9-a10d-2c75322e8a94">
